src/pages/Strategies/Strategies.tsx

```diff
@@ -18,14 +26,35 @@ const Strategies = (): JSX.Element => {
           transaction fees!
         </P>
       </Card>
-      <StyledList>
-        {Object.values(StrategyType).map((strategyType) => (
-          <StrategyCard key={strategyType} strategyType={strategyType} />
-        ))}
-        <Card alignItems='center' justifyContent='center'>
-          <P size='xs'>More Strategies coming soon</P>
-        </Card>
-      </StyledList>
+      {strategies ? (
+        <StyledList>
+          {strategies.map((strategy) => {
+            const { title, summary } = getContent(strategy, t);
+
+            const to = PAGES.STRATEGY.replace(`:${URL_PARAMETERS.STRATEGY.TYPE}`, strategy.type);
+
+            return (
+              <Link key={strategy.type} to={to}>
+                <StrategyCard
+                  risk={strategy.risk}
+                  ticker={strategy.currency.ticker}
+                  interestRate={strategy.interestRate}
+                  title={title}
+                  description={summary}
+                />
+              </Link>
+            );
+          })}
+          <Card alignItems='center' justifyContent='center'>
+            <P size='xs'>More Strategies coming soon</P>
+            <TextLink size='xs' underlined to={'#'}>
+              Request strategies
+            </TextLink>
```

Comment:
I would comment this out until we have a page to redirect to for the requests.

Reply:
Good catch, I will quick check with @nud3l if we already have the form link for this.